### ISSUE TITLE
How to install?

### ISSUE THREAD

**Issue by SrPeterr** [Manliketjb/chat#1]
Hi!
I do really like the design but Im having some problems installing it, should it replace the current chat resource?
If that is true, Im getting some errors and I cant even open the chat :/
![image](https://user-images.githubusercontent.com/21348463/156743940-110c707d-dc1b-4d24-8bab-948d020cece5.png)

**Comment by Manliketjb** [comment]
> Hi! I do really like the design but Im having some problems installing it, should it replace the current chat resource? If that is true, Im getting some errors and I cant even open the chat :/ ![image](https://user-images.githubusercontent.com/21348463/156743940-110c707d-dc1b-4d24-8bab-948d020cece5.png)

Yes it's meant to replace the chat but you need to keep the node_modules folder. i will be updating this chat within the next week

**Comment by Sheep-Dev** [comment]
> > Hi! I do really like the design but Im having some problems installing it, should it replace the current chat resource? If that is true, Im getting some errors and I cant even open the chat :/ ![image](https://user-images.githubusercontent.com/21348463/156743940-110c707d-dc1b-4d24-8bab-948d020cece5.png)
> 
> Yes it's meant to replace the chat but you need to keep the node_modules folder. i will be updating this chat within the next week

So i am using zap host do i rename this file qb-chat and just drop it over the other but keep node files

**Comment by Manliketjb** [comment]
> > > Hi! I do really like the design but Im having some problems installing it, should it replace the current chat resource? If that is true, Im getting some errors and I cant even open the chat :/ ![image](https://user-images.githubusercontent.com/21348463/156743940-110c707d-dc1b-4d24-8bab-948d020cece5.png)
> > 
> > 
> > Yes it's meant to replace the chat but you need to keep the node_modules folder. i will be updating this chat within the next week
> 
> So i am using zap host do i rename this file qb-chat and just drop it over the other but keep node files

I also use Zap. open the chat resource in the server files, delete the contents (not the file) of chat, except the node_modules folder. Then Drag the contents inside of my chat folder into the server chat folder
